docs/v.list

```diff
@@ -6,13 +6,13 @@
 
     <var name="kotlinVersion" value="2.0.20" type="string"/>
 
-    <var name="kotlinEapVersion" value="2.1.0-Beta1" type="string"/>
+    <var name="kotlinEapVersion" value="2.0.21-RC" type="string"/>
```

Comment:
Oh, I forgot that 2.1.0-Beta1 is more important than the new RC.
Let's rollback all the changes of this file.